.tekton/integration-tests/tasks/rhtap-install.yaml

```diff
@@ -35,5 +42,46 @@ spec:
         git clone "$(params.git-url)" .
         git checkout "$(params.git-revision)"
 
+        # Export the environment variables from the secret
+        cat /workspace/config.env > config.sh
+        chmod +x config.sh
+        source ./config.sh
+
+        # Path to your values.yaml.tpl file
+        tpl_file="rhtap-cli/charts/values.yaml.tpl"
+
+        # Create the new integrations section
+        new_integrations=$(cat <<EOF
+        integrations:
+          github:
+            id: "${GITHUB__APP__ID}"
+            clientId: "${GITHUB__APP__CLIENT__ID}"
+            clientSecret: "${GITHUB__APP__CLIENT__SECRET}"
+            publicKey: |
+        $(echo "${GITHUB__APP__PRIVATE_KEY}" | sed 's/^/      /')
+            token: "${GITOPS__GIT_TOKEN}"
+            webhookSecret: "${GITHUB__APP__WEBHOOK__SECRET}"
+          gitlab:
+            token: "${GITLAB__TOKEN}"
+          quay:
+            dockerconfigjson: |
+        $(echo "${QUAY__DOCKERCONFIGJSON}" | sed 's/^/      /')
+            token: "${QUAY__API_TOKEN}"
+        EOF
+        )
+
+        # Use awk to replace the integrations section
```

Comment:
I was looking at the test output of the last run and it doesn't look like these replaced values in the '# integrations:' section were applied

Link to most recent run as of today: https://console.redhat.com/preview/application-pipeline/workspaces/rhtap-shared/applications/rhtap-cli/pipelineruns/rhtap-cli-s5m25-pp8mf/logs?task=rhtap-install